src/platform/ESP32/ConnectivityManagerImpl.cpp

```diff
@@ -667,7 +667,7 @@ void ConnectivityManagerImpl::OnStationConnected()
     ChipDeviceEvent event;
     event.Type                          = DeviceEventType::kWiFiConnectivityChange;
     event.WiFiConnectivityChange.Result = kConnectivity_Established;
-    PlatformMgr().PostEvent(&event);
+    PlatformMgr().PostEventLoggingErrors(&event);
 
     UpdateInternetConnectivityState();
```

Comment:
Should this update happen even if the notification could not be posted?  Maybe file a followup to sort this out?  Similar for other things in this file.  I guess the existing error-handling here is pretty questionable....